### PR TITLE
Feature/show map pins

### LINKED ISSUE #4: As a fan when I open the map I expect to see events as pins on the map.
Data should be mocked as a JSON object on the client

### PR DESCRIPTION
**Pull request by GameKyuubi**
Resolves #4

### CODE REVIEW

File: src/Map.js [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+    onClick={props.onMapClick}
+  >
+    {props.markers.map((marker) => {
+      const position = {
+        lat: marker.lat,
+        lng: marker.lng,
+      };
+      return (<Marker
+        position={position}
+        key={position.lat + position.lng}
```
Comment (kumilange) [inline src/Map.js]: `position.lat + position.lng` could be the same. can we use concat instead of add?

**Review by kumilange — CHANGES_REQUESTED** [review]

File: src/Map.js (hunk 1 above)
Comment (GameKyuubi) [reply src/Map.js]: Good catch

**Comment by GameKyuubi** [comment]
Resolves #10

**Review by billfeng — APPROVED** [review]
LPGTM!

**Comment by kumilange** [comment]
@GameKyuubi LGTM!

**Review by kumilange — APPROVED** [review]

**Pull request merged**